pyproject.toml

```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "uid2_client"
-version = "2.4.6"
+version = "3.0.0a1"
```

Comment:
is this how the version should be bumped, or is it done as part of the pipeline?

Reply:
its done by the pipeline. It will bump up the major, minor or patch number corresponding to the Github action that's run. To publish an alpha package it will have to done manually